Makefile

```diff
@@ -132,26 +131,24 @@ logs-join:
 logs-clean:
 	-rm -f robottelo_gw*.log
 
-uuid-check:  ## list duplicated uuids
+uuid-check:  ## list duplicated or empty uuids
 	$(info "Checking for empty or duplicated @id: in docstrings...")
-	scripts/check_duplicate_uuids.sh
+	@scripts/check_uuids.sh
 
-uuid-replace-duplicate:  ## list duplicated uuids
-	scripts/replace_dup_uuids.sh
-
-uuid-replace-empty:  ## list duplicated uuids
-	scripts/replace_empty_uuids.sh
+uuid-fix:
+	@scripts/fix_uuids.sh
 
 flake8:
 	$(info "Checking style and syntax errors with flake8 linter...")
-	@flake8 . --show-source
+	@flake8 $(shell git diff --name-only) --show-source
```

Comment:
This is a good change, but I feel that it is more suitable for travis and commit hook other than checking local changes, I mean I will not be able to run `make flake8` for files that I have not added to git yet.